docs/docs/get_started/quickstart.mdx

```diff
@@ -31,22 +38,58 @@ import CodeBlock from "@theme/CodeBlock";
 
 For more details, see our [Installation guide](/docs/get_started/installation).
 
-### Environment
+### LangSmith
+
+Many of the applications you build with LangChain will contain multiple steps with multiple invocations of LLM calls.
+As these applications get more and more complex, it becomes crucial to be able to inspect what exactly is going on inside your chain or agent.
+The best way to do this is with [LangSmith](https://smith.langchain.com).
+
+Note that LangSmith is not needed, but it is helpful.
+If you do want to use LangSmith, after you sign up at the link above, make sure to set your environment variables to start logging traces:
+
+```shell
+export LANGCHAIN_TRACING_V2="true"
+export LANGCHAIN_API_KEY="..."
+```
+
+## Building with LangChain
+
+LangChain enables building application that connect external sources of data and computation to LLMs.
```

Comment:
I would break this up a little bit more into multiple lines - it looks long on

https://langchain-5o76madna-langchain.vercel.app/docs/get_started/quickstart